packages/loopring_v3/contracts/aux/bridge/Bridge.sol

```diff
@@ -440,17 +465,18 @@ contract Bridge is IBridge, BatchDepositor, Claimable
             for (uint i = 0; i < call.txGroups.length; i++) {
                 ConnectorTxGroup memory txGroup = call.txGroups[i];
                 for (uint j = 0; j < txGroup.transactions.length; j++) {
-                    ConnectorTx memory bridgeTx = txGroup.transactions[j];
+                    ConnectorTx memory connectorTx = txGroup.transactions[j];
                     deposits[txIdx++] = IBatchDepositor.Deposit({
-                        owner:  bridgeTx.owner,
-                        token:  bridgeTx.token,
-                        amount: bridgeTx.amount
+                        owner:  connectorTx.owner,
+                        token:  connectorTx.token,
+                        amount: connectorTx.amount
                     });
                 }
             }
             assert(txIdx == totalNumCalls);
-            emit ConnectorCalled(call.connector, false, returnData);
         }
+
+        emit ConnectorTransacted(call.connector, check, success, returnData);
```

Comment:
Do we care if the check failed? The connector call will be done as if it was successful, so it doesn't impact anything. It can't hurt to know, but I don't really know what we'd do with it. If it's false the connector is written badly which could be good to know, but seems like the connector needs to be tested better then. :)

Reply:
It is weird to me if we don't even care about the failure while still calling the `getMinGasLimit` method. If we are certain the call will not fail, then we should not call `getMinGasLimit`, otherwise, we'd better know it failed then later figure out why.

Reply:
The way I see it's like people doing a metatx (which these bridge transactions basically are, just with some batching on top of them). Do relayers care that something users try to do will fail? Not really. But the relayer does need to follow the rules the user expects. And one of these rules is these gas limits, which the relayer needs to folllow. 

So yes, we will make sure the call will be successful, but of course users can't just count on us doing so without just trusting us. But if it would not be successful (e.g. gas limit is exceeded to try to calculate the approximation) than that's fine too. If we didn't create the Connector it's not our problem to fix (like if a user calls some dapp using a metatx that fails, we don't have to care why it failed).

Reply:
We'll talk about it on zoom. Basically we want to answer ourselves this question: why we call `getMinGasLimit`? Potential answers are:

1. We call it and if it fails, then we promise our users not to proceed with the `processProcessorTransactions`.
2. We call it, but even if it fails, we proceed (users don't care), but we'll learn something from the failure and optimize future gas calculations offchain.

I would say we chose 2.